html/semantics/document-metadata/the-link-element/link-style-error-quirks.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE HTML PUBLIC "-//Sun Microsystems Corp.//DTD HotJava Strict HTML//" "">
```

Comment:
Can you replace this with a comment saying "quirks mode"?

Reply:
That's a lot less fun.